app/controllers/actions/asset/ProvisionUtil.scala

```diff
@@ -246,7 +246,7 @@ trait Provisions extends ProvisionUtil with AssetAction { self: SecureAction =>
     import play.api.Play.current
 
     val ActionDataHolder(asset, pRequest, _, attribs) = adh
-    BackgroundProcessor.send(ProvisionerTest(pRequest)(request)) { res =>
+    BackgroundProcessor.send(ProvisionerTest(pRequest)(request, defaultContext)) { res =>
```

Comment:
Yes, only because the second parameter list for implicits is being manually invoked because 'request' is not an implicit value.